src/components/BooksForm.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+export default function BooksForm() {
+  const categories = ['Action', 'Biography', 'History', 'Horror', 'Kids', 'Learning', 'Sci-Fi'];
+  return (
+    <div>
+      <form action="">
+        <input type="text" />
+        <input type="text" />
```

Comment:
- [ ] The category is going to be inputted by the select tag, there is no need to have an input tag for it.